lsp-pyls.el

```diff
@@ -290,6 +290,54 @@ at all."
   :group 'lsp-pyls
   :package-version '(lsp-mode . "6.1"))
 
+(defcustom lsp-pyls-plugins-flake8-enabled nil
+  "Enable or disable the plugin."
+  :type 'boolean
+  :group 'lsp-pyls
+  :package-version '(lsp-mode . "6.1"))
```

Comment:
these should be 6.2

Reply:
I think this should be 6.3. We bumped the version a few weeks ago.

Reply:
My understanding is that we are 6.2 until we tag release - https://github.com/emacs-lsp/lsp-mode/releases . I am not sure when we should bump the package header version - when we start new release when we release it. I am fine with both approaches. But still, if we tag 6.2 we should update the changelog for the release :).

Reply:
updated to 6.2 )